template.yaml

```diff
@@ -0,0 +1,337 @@
+                type: string
+              displayName:
+                type: string
+              shortName:
+                type: string
+              archiveName:
+                type: string
+              cname:
+                type: string
+              institutionDns:
+                type: string
+              administrationId:
+                type: string
+                format: email
+              feideOrganizationId:
+                type: string
+          Problem:
+            type: object
+            properties:
+              title:
+                type: string
+              status:
+                type: integer
+              detail:
+                type: string
+        securityDefinitions:
+          CognitoUserPool:
+            type: apiKey
+            name: Authorization
+            in: header
+            x-amazon-apigateway-authtype: cognito_user_pools
+            x-amazon-apigateway-authorizer:
+              type: cognito_user_pools
+              providerARNs:
+                - !Ref CognitoAuthorizerArn
+  NvaCreateCustomerFunction:
+    Type: AWS::Serverless::Function
+    Properties:
+      CodeUri: create-customer
+      Handler: no.unit.nva.customer.create.CreateCustomerHandler::handleRequest
+      Runtime: java11
+      MemorySize: 1400
+      Environment:
+        Variables:
+          ALLOWED_ORIGIN: '*'
+          TABLE_NAME: !Ref CustomersTableName
+      Policies:
+        - DynamoDBCrudPolicy:
+            TableName: !Ref CustomersTableName
+      Events:
+        PostEvent:
+          Type: Api
+          Properties:
+            Path: /
+            Method: post
+            RestApiId: !Ref NvaCustomerApi
+  NvaGetCustomerFunction:
+    Type: AWS::Serverless::Function
+    Properties:
+      CodeUri: get-customer
+      Handler: no.unit.nva.customer.get.GetCustomerHandler::handleRequest
+      Runtime: java11
+      MemorySize: 1400
+      Environment:
+        Variables:
+          ALLOWED_ORIGIN: '*'
+          TABLE_NAME: !Ref CustomersTableName
+      Policies:
+        - DynamoDBCrudPolicy:
+            TableName: !Ref CustomersTableName
+      Events:
+        GetEvent:
+          Type: Api
+          Properties:
+            Path: /{identifier}
+            Method: get
+            RestApiId: !Ref NvaCustomerApi
+  NvaGetAllCustomersFunction:
+    Type: AWS::Serverless::Function
+    Properties:
+      CodeUri: get-all-customers
+      Handler: no.unit.nva.customer.getall.GetAllCustomersHandler::handleRequest
+      Runtime: java11
+      MemorySize: 1400
+      Environment:
+        Variables:
+          ALLOWED_ORIGIN: '*'
+          TABLE_NAME: !Ref CustomersTableName
+      Policies:
+        - DynamoDBCrudPolicy:
+            TableName: !Ref CustomersTableName
+      Events:
+        GetEvent:
+          Type: Api
+          Properties:
+            Path: /
+            Method: get
+            RestApiId: !Ref NvaCustomerApi
+  NvaUpdateCustomerFunction:
```

Comment:
Recommendation generated by Amazon CodeGuru Reviewer. Leave feedback on this recommendation by replying to the comment or by reacting to the comment using emoji.

You must specify a value that is greater than or equal to 128, and it must be a multiple of 64. You cannot specify a size larger than 3008. Error at Resources/NvaUpdateCustomerFunction/Properties/MemorySize